app/models/manageiq/providers/amazon/manager_mixin.rb

```diff
@@ -76,7 +76,8 @@ def verify_credentials(args)
       # Pull out the password from the database if a provider ID is available
       secret_access_key ||= find(args["id"]).authentication_password('default')
 
-      !!raw_connect(access_key, secret_access_key, :EC2, region, proxy_uri, validate = true, :assume_role => service_account)
+      connection = raw_connect(access_key, secret_access_key, :EC2, region, proxy_uri, :assume_role => service_account)
+      !!validate_connection(connection)
```

Comment:
Future refactoring but I think we can remove the `verify` logic from `raw_connect` now since that looked like the only caller (we should verify it was the only caller with verify=true)